### PR TITLE
Delegate contract updates

### PR DESCRIPTION
**Pull request by 0xldr**
Actions changes from our review on this PR: https://github.com/makerdao/mips/pull/1120

To be merged once the Governance Scope is updated to the same effect.

Also removes all avatar files to improve consistency, some ADs have avatars, other's don't. AVC avatars are outdated.

### CODE REVIEW

File: governance/delegates/0x3C85C3F4B0d610Fd29cc132A2073f85C15F83f2bvv/profile.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+name: WBC
+external_profile_url: https://forum.makerdao.com/t/wbc-aligned-delegate-communications/20828
+---
+
+# WBC
+Delegate Address: 0x3C85C3F4B0d610Fd29cc132A2073f85C15F83f2bvv
```
Comment (boet1) [inline governance/delegates/0x3C85C3F4B0d610Fd29cc132A2073f85C15F83f2bvv/profile.md:7]: The delegate's address has two extra 'v'. Additionally, the name of the folder containing it has the same address with the same error. I have reviewed the transactions, and this delegate created this contract: 0xc139042755eae94bd10798251801b5bd26464b69. 
I believe the folder should have that name. Could you also check it to be sure, as the error is repeated in the MIPs? Thank you.
`Delegate Address: 0x3C85C3F4B0d610Fd29cc132A2073f85C15F83f2b`

File: governance/delegates/0x5C86FA06eEbF5D7bdc1a75c293a3289AC0535F7F/profile.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+---
+name: Skynet
+external_profile_url: https://forum.makerdao.com/t/skynet-ad-submission/21077
+---
+
+# Skynet
+Delegate Address: 0xad8b61d293d362bbff12db6331ecfca8a7221d5b
```
Comment (boet1) [inline governance/delegates/0x5C86FA06eEbF5D7bdc1a75c293a3289AC0535F7F/profile.md:7]: `Delegate Address: 0xd4d1A446cD5976a11bd32D3e815A9F85FED2F9F3`

File: governance/delegates/0x3C85C3F4B0d610Fd29cc132A2073f85C15F83f2bvv/profile.md (hunk 1 above)
Comment (boet1) [reply governance/delegates/0x3C85C3F4B0d610Fd29cc132A2073f85C15F83f2bvv/profile.md:7]:
```suggestion
Delegate Address: 0x3C85C3F4B0d610Fd29cc132A2073f85C15F83f2b
```

File: governance/delegates/0x5C86FA06eEbF5D7bdc1a75c293a3289AC0535F7F/profile.md (hunk 2 above)
Comment (boet1) [reply governance/delegates/0x5C86FA06eEbF5D7bdc1a75c293a3289AC0535F7F/profile.md:7]:
```suggestion
Delegate Address: 0xd4d1A446cD5976a11bd32D3e815A9F85FED2F9F3
```

File: governance/delegates/0x3C85C3F4B0d610Fd29cc132A2073f85C15F83f2bvv/profile.md (hunk 1 above)
Comment (0xldr) [reply governance/delegates/0x3C85C3F4B0d610Fd29cc132A2073f85C15F83f2bvv/profile.md:7]: https://github.com/makerdao/mips/blob/master/MIP113/MIP113.md?plain=1#L374

It is correct in the MIP (https://github.com/makerdao/mips/blob/master/MIP113/MIP113.md?plain=1#L374) we think this is just a copy-paste artefact. Good spot.

**Review by VoteWizard — APPROVED** [review]
Thanks, ldr!

**Pull request merged**